src/main/java/SpringJPA/Model/User.java

```diff
@@ -29,7 +33,18 @@ public User(String user, String pass, String role, String mail) {
         this.password = pass;
         this.role = role;
         this.email = mail;
+    }
 
+    public User(String user, String pass, String role, long apiCallCount, long apiCallLimit) {
+        this.username = user;
+        this.password = pass;
+        this.role = role;
+        this.apiCallCount = apiCallCount;
+        this.apiCallLimit = apiCallLimit;
+    }
+
+    public void incrementApICallCount(){
+        this.apiCallCount++;
```

Comment:
Should there be a maximum call count for APIs?

Reply:
yes, you are correct, I will implement that change now

Reply:
I have made the change, please review